src/AutoRemoteSyscalls.cc

```diff
@@ -588,7 +588,7 @@ void AutoRemoteSyscalls::finish_direct_mmap(
    * recording. */
   {
     AutoRestoreMem child_str(*this, backing_file_name.c_str());
-    fd = infallible_syscall(syscall_number_for_open(arch()),
+    fd = infallible_syscall(syscall_number_for_openat(arch()), -1,
```

Comment:
I guess passing -1 here works because the path is always absolute?

Reply:
Yes, this is the intended way to open paths that are guaranteed to be absolute.

Reply:
Makes sense. I thought this was AT_FDCWD at first and had to look up its value :)